objects.js

```diff
@@ -0,0 +1,70 @@
+`use strict`;
+// Objects 1. BetweenDates
+
+function durationBetweenDates (startDate = 'now', endDate = 'now', dimension = 'seconds') {
+    const units = {
+      days: 86400000,
+      hours: 3600000,
+      minutes: 60000,
+      seconds: 1000
+    };
+    
+    const startTime = Date.parse(startDate);
+    const endTime = Date.parse(endDate);
+    
+    const diff = Math.abs(startTime - endTime);
+    const result = Math.floor(diff / units[dimension]);
+    
+    return `${result} ${dimension}`;
+  }
+  
+  console.log(durationBetweenDates('21 Jul 2025', '19 Mar 2022', 'seconds'));
```

Comment:
<img width="450" alt="Screenshot 2023-03-24 at 12 00 18" src="https://user-images.githubusercontent.com/1708513/227490654-303218d1-4426-41f8-b2c3-b832779aa764.png">

NaN is wrong answer. Something wrong with default values

Reply:
Hmm, but you have no dates in the console log. If you specify the dates, everything will work, won't it? Or have I misunderstood something in the task?